composer.json

```diff
@@ -44,7 +44,7 @@
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<2.1 || >=3.0",
-        "jms/serializer": "<0.13 || >=3.0",
+        "jms/serializer": "<1.3 || >=4.0",
```

Comment:
title says "Allow jms serializer ^3.0" but this change makes it impossible to install serializer 3.0 https://jubianchi.github.io/semver-check/#/%3C1.3%20||%20%3E%3D4.0/3.0.0 or am I missing something?

Reply:
This constraint is at the "conflict" section, it disallows versions lower to `1.3.0` and higher or equal than `4.0.0`.

Reply:
ah, sry, didn't see it was in conflict